addons/ace_interact/fnc_radioListChildrenActions.sqf

```diff
@@ -46,9 +46,22 @@ private _radioList = [] call EFUNC(api,getCurrentRadioList);
     private _picture = getText (_item >> "picture");
     private _isActive = _x isEqualTo _currentRadio;
 
-    private _action = [_x, _displayName, _picture, {}, {true}, {_this call FUNC(radioChildrenActions)}, [_x, _isActive, _pttAssign]] call ace_interact_menu_fnc_createAction;
+    private _action = [
+        _x,
+        _displayName,
+        _picture,
+        {
+            [(_this select 2) select 0] call EFUNC(sys_radio,openRadio)
+        },
+        {
+            (_this select 2) params ["_radio"];
+            !((_radio in ACRE_ACCESSIBLE_RACK_RADIOS && {isTurnedOut acre_player}) || (toLower _radio) in ACRE_HEARABLE_RACK_RADIOS)
```

Comment:
This is not necessary since openRadio handles all these cases already. It even shows a hint :)